docs/configuration.rst

```diff
@@ -291,13 +291,13 @@ below:
 
          Overrides ``DD_TRACE_PROPAGATION_STYLE`` for injection propagation style.
 
-         The supported values are ``datadog``, ``b3``, ``b3 single header``, and ``none``.
+         The supported values are ``datadog``, ``b3multi``, and ``b3 single header``, and ``none``.
 
          All provided styles are injected into the headers of outbound requests.
-
+         
```

Comment:
Is this white space expected

Reply:
yes

Reply:
Perfect. Approved

Reply:
I don't have a preference, I'll remove it.